src/Hintman.hs

```diff
@@ -2,5 +2,29 @@ module Hintman
        ( someFunc
        ) where
 
+import Options.Applicative
+
+-- | Represents the application settings
+data Context = Context
+    { contextLogging :: Bool -- ^ If True, enable logging
+    }
+
+parseContext :: Parser Context
+parseContext = Context
+    <$> switch
```

Comment:
Probably we will have some other cli options in the future, so I guess it makes sense to move logging parser to the separate function (`loggingP` or any other name)